index.html

```diff
@@ -2,9 +2,9 @@
 <html>
     <head>
         <meta charset="utf-8" />
-        <meta http-equiv="refresh" content="0; url=https://join.slack.com/t/crossplane/shared_invite/zt-26kxm0k2a-esMwXvMpfw8glyoMzukAlQ" />
+        <meta http-equiv="refresh" content="0; url=https://join.slack.com/t/crossplane/shared_invite/zt-29oiqovki-WuHqY~LUaAIHZVChxJNUqg" />
```

Comment:
perfect! this link works for me and is set correctly to "never expire" as per slack admin UI ✅ 

<img width="1080" alt="Screenshot 2023-12-28 at 9 15 09 AM" src="https://github.com/crossplane/slack.crossplane.io/assets/4313439/7821328c-6b55-451f-ba28-2a2aabfeec21">